tomviz/python/BinaryThreshold.py

```diff
@@ -68,11 +66,12 @@ def transform_scalars(self, dataset, lower_threshold=40.0,
             self.progress.message = "Creating child data set"
 
             # Set the output as a new child data object of the current data set
-            label_map_dataset = vtkImageData()
-            label_map_dataset.CopyStructure(dataset)
+            label_map_dataset = dataset.create_child_dataset()
+
+            result = itk.GetArrayFromImage(threshold_filter.GetOutput())
+            # Transpose the data to Fortran indexing
+            label_map_dataset.active_scalars = result.transpose([2, 1, 0])
```

Comment:
Before, `itkutils.set_array_from_itk_image()` would call `utils.set_array()` with `isFortran=False` as an argument. This argument indicated that the indexing needed to be swapped from (i, j, k) to (k, j, i). This is done for ITK outputs because ITK uses C indexing, whereas we use Fortran indexing for tomviz.

Now, we are just calling `result.transpose([2, 1, 0])` to swap the indexing within the operator so that tomviz receives the data with the correct indexing.

Note that Fortran indexing vs. C indexing is not the same as Fortran vs C memory ordering.

Reply:
We moved this logic to a new `itkutils` function.